examples/http3-client.c

```diff
@@ -100,7 +100,7 @@ static void flush_egress(struct ev_loop *loop, struct conn_io *conn_io) {
 
 static int for_each_setting(uint64_t identifier, uint64_t value,
                            void *argp) {
-    fprintf(stderr, "got HTTP/3 SETTING: %lu=%lu\n",
+    fprintf(stderr, "got HTTP/3 SETTING: %" PRId64 "=%" PRId64 "\n",
```

Comment:
These values are unsigned, so I think we need `PRIu64` here?
```suggestion
    fprintf(stderr, "got HTTP/3 SETTING: %" PRIu64 "=%" PRIu64 "\n",
```

Reply:
Fixed. Thanks